content/references/rippled-api/api-conventions/basic-data-types.md

```diff
@@ -81,76 +81,27 @@ If you do not specify a ledger, the `current` (in-progress) ledger is chosen by
 
 
-## Currencies
+## Specifying Currency Amounts
 
-There are two kinds of currencies in the XRP Ledger: XRP, and everything else. There are many differences between the two:
+There are two kinds of currencies in the XRP Ledger: XRP, and issued currencies. These two types of currencies are specified in different formats, and they have different amounts of precision and different rounding behavior.
 
-| `XRP`                                                           | Issued Currencies |
-|:----------------------------------------------------------------|:-----------|
-| Has no issuer.                                                  | Always issued by an XRP Ledger account |
-| Specified as a string                                           | Specified as an object |
-| Tracked in [accounts](accountroot.html) | Tracked in [trust lines](ripplestate.html) |
-| Can never be created; can only be destroyed                     | Can be issued or redeemed freely |
-| Maximum value `100000000000` (`1e11`)                           | Maximum value `9999999999999999e80` |
-| Precise to the nearest ["drop"](#xrp) (0.000001 XRP)            | 15 decimal digits of precision, with a minimum nonzero absolute value of `1000000000000000e-96` |
+Some fields, such as the destination `Amount` of a [Payment transaction][], can be either type. Some fields only accept XRP specifically, such as the `Fee` field ([transaction cost](transaction-cost.html)).
 
-**Caution:** The XRP Ledger uses decimal math with different precision than typical floating-point numbers, so currency amounts are always presented as strings.
+XRP is specified as a string containing an integer number of "drops" of XRP, where 1 million drops equals 1 XRP. Issued currencies are instead specified as an object with fields for the decimal amount, currency code, and issuer. For example:
 
-### Specifying Currency Amounts
+- **XRP** - To specify an `Amount` field with a value of 13.1 XRP:
 
-Some API methods require you to specify an amount of currency. Depending on whether you are dealing in the network's native XRP currency or other currency units (called _issuances_), the style for specifying it is very different.
+        "Amount": "13100000"
 
-#### XRP
-[drops of XRP]: #xrp
-[XRP, in drops]: #xrp
+- **Issued Currency** - To specify an `Amount` field with a value of 13.1 FOO issued by or to rf1B...:
 
-Amounts of XRP are represented as strings. (XRP has precision equivalent to a 64-bit integer, but JSON integers are limited to 32 bits, so XRP can overflow if represented in a JSON integer.) XRP is formally specified in "drops", which are equivalent to 0.000001 (one 1-millionth) of an XRP each. Thus, to represent 1.0 XRP in a JSON document, you would write:
+        "Amount": {
+            "value": "13.1",
+            "code": "FOO",
+            "issuer": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn"
```

Comment:
Second question first, we don't have a list.

rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn is an address I control. I use it out of convenience when showcasing examples from the Mainnet. I have another address where I keep most of my XRP but I do use rf1B(..) for various stuff.

Partly that's because of @justmoon's funny anecdote about how he used his real Bitcoin address as the sample destination in the docs for his early Bitcoin JS client and received a surprising amount of money from people who didn't both changing the examples before running them. So, the upside of using your own address in examples is some people might send you free money. The downside is people have also copied example code setting their account's `Domain` field to match mine, so I occasionally get questions like, "Hey, why does this suspicious address say it's owned by you?" 😹 

On the Devnet and Testnet, I generally use the faucet address [rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe](https://testnet.xrpl.org/accounts/rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe) as the default destination for payments so by default the Test XRP gets added back to the supply. For anything else, you can just pull a new address from the faucet as needed and throw it away afterwards; addresses are as numerous and disposable as grains of sand.

Another address you may see in examples is r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59. I don't know whose that one is; it predates my work at Ripple. I think it probably belongs to a former Ripple employee, so we should not use that one in new examples.

Occasionally I'll also use the public issuing addresses of longstanding XRPL businesses like Bitstamp if I need an example of a very active address with a lot going on. I try not to use other random addresses if I don't know who runs them, since I feel like that could direct unwanted attention to people who didn't ask for it. Sure, all addresses and transactions on the XRP Ledger are public information, but it feels rude to just pick specific addresses out of the crowd without permission and draw attention to them.

So, feel free to use your own address, or mine, or generate new addresses as needed.